### PR TITLE
ref/remove_resources_bundle

### PR DESCRIPTION
**Pull request by ritamsarmah**
## Ticket Link
https://app.asana.com/0/0/1207946837689465/f

## Description

Removes separate inclusion of SDK bundle

### CODE REVIEW

File: Sources/AppLovinSDKResources/AppLovinSDKResources.swift [hunk 1]
```diff
@@ -0,0 +1,9 @@
+//
+//  AppLovinSDKResources.swift
+//  AppLovinSDK
+//
+//  Created by Ritam Sarmah on 8/2/24.
+//  Copyright © 2024 AppLovin Corporation. All rights reserved.
+//
+
+// Empty file for SPM wrapper target
```
Comment (harryarakkal) [inline Sources/AppLovinSDKResources/AppLovinSDKResources.swift:9]: For my understanding, is this just to keep the `AppLovinSDKResources` target with all the linker settings?

**Review by harryarakkal — APPROVED** [review]
To confirm, you've tested this new integration works right?

File: Sources/AppLovinSDKResources/AppLovinSDKResources.swift (hunk 1 above)
Comment (ritamsarmah) [reply Sources/AppLovinSDKResources/AppLovinSDKResources.swift:9]: Yep exactly

**Comment by ritamsarmah** [comment]
> To confirm, you've tested this new integration works right?

@harryarakkal Yeah, I should've mentioned—I tested by pointing to this specific branch in a test app and loading a few ads.

**Pull request merged**